x-pack/legacy/plugins/graph/public/plugin.ts

```diff
@@ -34,18 +35,18 @@ export class GraphPlugin implements Plugin {
   private savedObjectsClient: SavedObjectsClientContract | null = null;
   private angularDependencies: LegacyAngularInjectedDependencies | null = null;
 
-  setup(core: CoreSetup, { __LEGACY: { xpackInfo, Storage } }: GraphPluginSetupDependencies) {
+  setup(core: CoreSetup, { __LEGACY: { Storage }, licensing }: GraphPluginSetupDependencies) {
     core.application.register({
       id: 'graph',
       title: 'Graph',
       mount: async ({ core: contextCore }, params) => {
```

Comment:
you could migrate await core.getStartServices() ... or take care of this in a future PR